back/engines/commercial/content_builder/app/services/content_builder/side_fx_layout_service.rb

```diff
@@ -2,13 +2,17 @@ module ContentBuilder
   class SideFxLayoutService
     include SideFxHelper
 
-    def before_create(layout, user); end
+    def before_create(layout, _user)
+      layout.craftjs_jsonmultiloc = LayoutImageService.new.swap_data_images layout, :craftjs_jsonmultiloc
```

Comment:
Is there a test for this change? Same question for the change in `before_update`.